lib/big_brother/cluster.rb

```diff
@@ -1,6 +1,15 @@
 module BigBrother
   class Cluster
-    attr_reader :fwmark, :scheduler, :check_interval, :nodes, :name, :ramp_up_time, :nagios
+    attr_reader :fwmark, :scheduler, :check_interval, :nodes, :name, :ramp_up_time, :nagios, :backend_mode
+    ACTIVE_PASSIVE_CLUSTER = "active_passive"
+
+    def self.create_cluster(name, attributes)
```

Comment:
Why is this method necessary? Why couldn't you add this functionality to the existing initialize method?